packages/react-styled-ui/src/Badge/index.js

```diff
@@ -14,7 +14,7 @@ const Badge = forwardRef(
         textTransform="uppercase"
```

Comment:
I sugges leave `textTransform` prop empty, so the frontend developer can choose the text case by themself.

![image](https://user-images.githubusercontent.com/447801/79939241-2f7bfa80-8491-11ea-8c73-acc943ee464d.png)